lti_consumer/outcomes.py

```diff
@@ -162,15 +162,16 @@ def handle_request(self, request):
             'imsx_messageIdentifier': 'unknown',
             'response': ''
         }
+        request_body = request.body.decode('utf-8')
```

Comment:
I believe we should do it in `except` block where we are escaping it, because it might break something else while parsing it.

Reply:
i did it intentionally because as per the docstring of `parse_grade_xml_body` it is expecting string object while our codebase is sending it bytestring.I think it will not break anything because `request.body` is not being used anywhere else in the method.